rviz_default_plugins/src/rviz_default_plugins/displays/depth_cloud/depth_cloud_display.cpp

```diff
@@ -370,6 +404,7 @@ void DepthCloudDisplay::subscribe()
           std::bind(&DepthCloudDisplay::processDepthMessage, this, std::placeholders::_1));
       }
     }
+    subscription_start_time_ = rviz_ros_node_->get_raw_node()->now();
```

Comment:
Are we not using / exposing this variable anywhere else ? Could be removed otherwise.

Reply:
I created a new PR to fix the crash https://github.com/ros2/rviz/pull/1161